Source/ocdm/open_cdm_impl.h

```diff
@@ -80,7 +80,9 @@ struct OpenCDMAccessor : public OCDM::IAccessorOCDM {
             ASSERT(_remote != nullptr);
 
             if (_remote == nullptr) {
-                _client.Release();
+                if (_client.IsValid()) {
```

Comment:
Think it is better if do this check before accessing the _client instance, i.e, before calling _client->IsOpen()